cypress/integration/multi_select_spec.js

```diff
@@ -0,0 +1,142 @@
+context('Single Select', () => {
+    beforeEach(() => {
+      cy.visit('./examples/MultiSelect/index.html')
+    })
+  
+    // describe('disabled', () => {
+    //   it('cannot be opened by clicking on it', () => {
+    //     // disable select first
+    //   })
+    // })
+  
+    describe('enabled', () => {
+      it('can be opened by clicking on it', () => {
+        cy.get('#smart-select-component')
+          .click()
+          .should('have.class', 'elm-smart-select--enabled-opened')
+      })
+    
+      it('can be closed by clicking outside of it', () => {
+        cy.get('#smart-select-component')
+          .click()
+          .should('have.class', 'elm-smart-select--enabled-opened')
+    
+        cy.get('#form-submission-status').click()
+    
+        cy.get('#smart-select-component')
+          .should('have.class', 'elm-smart-select--enabled-closed')
+      })
+    
+      it('can filter options via search', () => {
+        cy.get('#smart-select-component')
+          .click()
+        
+        cy.get('#elm-smart-select--select-options-container')
+          .children()
+          .should('have.length', 9)
+  
+        cy.get('#smart-select-input')
+          .type('3')
+  
+        cy.get('#elm-smart-select--select-options-container')
+          .children()
+          .should('have.length', 1)
+      })
+  
+      // it('shows an error message if no search results are found', () => {
+      //   cy.get('#smart-select-component')
+      //     .click()
+      // })
+  
+      it('can navigate options with \'Up\' and \'Down\' arrow keys', () => {
+        cy.get('#smart-select-component')
+          .click()
+  
+        cy.get('#elm-smart-select--select-options-container')
+          .children()
+          .first()
+          .should('have.class', 'elm-smart-select--select-option-focused')
+  
+        cy.get('#smart-select-component')
+          .type('{downarrow}')
+  
+        cy.get('#option-1')
+          .should('have.class', 'elm-smart-select--select-option-focused')
+
+        cy.get('#smart-select-component')
+          .type('{uparrow}')
+
+        cy.get('#option-0')
+          .should('have.class', 'elm-smart-select--select-option-focused')
+      })
+  
+      it('can be closed by pressing \'Escape\'', () => {
+        cy.get('#smart-select-component')
+          .click()
+          .should('have.class', 'elm-smart-select--enabled-opened')
+          .type('{esc}')
+          .should('have.class', 'elm-smart-select--enabled-closed')
+      })
+    
+      it('can select multiple options by clicking on them', () => {
+        cy.get('#smart-select-component')
+          .click()
+  
+        cy.get('#elm-smart-select--select-options-container')
+          .children()
+          .first()
+          .click()
+          .click()
+
+        cy.get('.elm-smart-select--multi-selected-container')
+          .children()
+          .first()
```

Comment:
These three lines seem to appear quiet frequently, so you could extract them into a helper or command function to improve readability